pyteal/ast/gitxn_test.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from pyteal.ast.gitxn import Gitxn, GitxnExpr, GitxnaExpr, InnerTxnGroup
```

Comment:
Similar comment here, these should be imported by `pyteal/ast/__init__.py` in order to have them be publicly exported.

The reason to avoid internal imports as much as possible in tests is so that we can be sure we're testing features in the same way that developers will use them

Reply:
good catch.